packages/libs/components/src/components/plotControls/MultiSelection.tsx

```diff
@@ -0,0 +1,80 @@
+import { useState, useEffect } from 'react';
+import { createStyles, makeStyles, Theme } from '@material-ui/core/styles';
+import Input from '@material-ui/core/Input';
+import InputLabel from '@material-ui/core/InputLabel';
+import MenuItem from '@material-ui/core/MenuItem';
+import FormControl from '@material-ui/core/FormControl';
+import ListItemText from '@material-ui/core/ListItemText';
+import Select from '@material-ui/core/Select';
+import Checkbox from '@material-ui/core/Checkbox';
+// import Chip from '@material-ui/core/Chip';
+
+interface MultiSelectionProps {
```

Comment:
This looks very nice! We expect to have thousands of nodes so i think it'll be very crucial to have a search function in the select box as well. After looking around a bit i found https://react-select.com/home which has a searchable, multi-select component. Do you think that would work fine here? I think it has all the functionality we need!

Reply:
@asizemore as mentioned at other place, now the node label control uses React-Select, not MUI